Assets/MixedRealityToolkit-Examples/Demos/Boundary/Materials.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
This meta doesn't have a corresponding folder.

![image](https://user-images.githubusercontent.com/3580640/42978769-8c45c6be-8b83-11e8-8c03-386b807645c0.png)

Reply:
missed deleting this after I moved files around..